packages/core-modules/contracts/submodules/election/ElectionBase.sol

```diff
@@ -31,6 +31,7 @@ contract ElectionBase is ElectionStorage, InitializableMixin {
     error NotNominated();
     error NoCandidates();
     error NoVotePower();
+    error HasNotVoted();
```

Comment:
Not sure about this, but this error looks different to the other ones, maybe something like `VoteNotCasted`, `NoCastedVote`

Reply:
Ok, will rename to  `VoteNotCasted`.